Makefile

```diff
@@ -194,6 +194,21 @@ antctl: $(ANTCTL_BINARIES)
 antctl-release:
 	@$(GO) build -o $(BINDIR)/$(ANTCTL_BINARY_NAME) $(GOFLAGS) -ldflags '-s -w $(LDFLAGS)' antrea.io/antrea/cmd/antctl
 
+.PHONY: check-copyright
+check-copyright: 
+	@echo "==> Installing addlicense <=="
+	@$(GO) install github.com/google/addlicense@latest
+	@echo "===> Checking License for files <==="
+	addlicense -c "Antrea Authors." -check `find . -type f -name "*.go"` `find . -type f -name "*.sh"`
```

Comment:
This doesn't seem to work for me:
```
$ make check-copyright
==> Installing addlicense <==
===> Checking License for files <===
addlicense -c "Antrea Authors." -check `find . -type f -name "*.go"` `find . -type f -name "*.sh"`
/bin/bash: addlicense: command not found
make: *** [check-copyright] Error 127
```

Reply:
@antoninbas  I cannot reproduce this error when everything is set up. May I know if the GOPATH is added to $PATH or if the GO Version is 1.17? 
Now I move some parts to a shell script, can this be tried again? if failed, I have the details of `make check-copyright -d` for debugging and the OS platform?

Reply:
It's better not to assume that `GOPATH` is in the PATH. You can look at what we did for other go tools: https://github.com/antrea-io/antrea/blob/main/hack/verify-spelling.sh

Reply:
@antoninbas changed